openlibrary/core/models.py

```diff
@@ -1116,6 +1116,17 @@ def get_default_cover(self):
                 return Image(web.ctx.site, "b", cover_id)
 
 
+# TODO: expand on tag model
+class Tag(Thing):
+    """Class to represent /type/tag objects in OL."""
+
+    def url(self, suffix="", **params):
```

Comment:
In Python these are usually called `kwargs` for keyword arguments.
```suggestion
    def url(self, suffix="", **kwargs):
```

Reply:
hmm, seems like the other classes (e.g. Lists) in `core/models.py` uses `**params` though. Should I rename the other classes to use `**kwargs` as well for uniformity?